repo/packages/B/beta-confluent-kafka/0/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Apache Kafka by Confluent\n\n\tDocumentation: https://www.confluent.io/whitepaper/deploying-confluent-platform-with-mesosphere",
+  "framework": true,
+  "licenses": [
+    {
+      "name": "Apache License v2",
+      "url": "https://raw.githubusercontent.com/confluentinc/kafka/trunk/LICENSE"
+    }
+  ],
+  "maintainer": "partner-support@confluent.io",
+  "minDcosReleaseVersion": "1.8",
+  "name": "beta-confluent-kafka",
+  "packagingVersion": "3.0",
```

Comment:
Like this: https://github.com/mesosphere/universe/blob/b6a7c7a136c59453fbe6cc26aceecdfe9d54486f/repo/packages/B/beta-dse/0/package.json#L10